tests/services/data.service.spec.ts

```diff
@@ -0,0 +1,156 @@
+import { of } from 'rxjs';
+import { DataService } from '../../src/app/core/services/data.service';
+import { GithubService } from '../../src/app/core/services/github.service';
+import { autoSpy } from '../auto-spy';
+import { csvString, jsonData, dataFileTeamStructure } from '../constants/data.constants';
+
+describe('DataService', () => {
+  it('when getDataFile is called it should return the correct data', done => {
+    const { build, githubService } = setup();
+    const dataService = build();
+    githubService.fetchDataFile.and.returnValue(of({'data': csvString}));
+
+    dataService.getDataFile().subscribe(actual => {
+      expect(actual).toEqual(jsonData);
+      done();
+    });
+  });
+
+  it('when getDataFile is called it should set dataFile with correct data', done => {

```

Comment:
Perhaps both the specs that test `getDataFile` method can be grouped in a single describe block?
So it might look like:

```js
describe('DataService', () => {
  describe('.getDataFile()', () => {
    it('returns the correct json data', () => {
       ... test code ...
    });
    it('sets dataFile with correct team structure data', () => {
       ... test code ...
   });
  });
... other tests ...

```